osu.Game.Rulesets.Osu/OsuInputManager.cs

```diff
@@ -39,6 +40,17 @@ protected override bool Handle(UIEvent e)
             return base.Handle(e);
         }
 
+        protected override bool HandleMouseTouchStateChange(TouchStateChangeEvent e)
+        {
+            if (!AllowUserCursorMovement)
+            {
+                // Still allow for forwarding of the "touch" part, but block the positional data.
+                e = new TouchStateChangeEvent(e.State, e.Input, e.Touch, false, null);
```

Comment:
I think this makes more sense rather than forcing `IsActive` to false, which sounded wrong but apparently doesn't matter on base method:
```csharp
                var touch = new Touch(e.Touch.Source, CurrentState.Mouse.Position);
                e = new TouchStateChangeEvent(e.State, e.Input, touch, e.IsActive, null);
```

Reply:
Is there a reason this change hasn't been applied @peppy ?

Reply:
I missed it. I think it looks fine as long as you test it to work as expected. Not 100% sure on the transfer of position from the mouse like that.

Reply:
Yeah well, it's the only way to make it work in that direction, and I think it's fine personally as you're passing the current state of the mouse position, ignoring the position passed by the touch.